relayer/clients.go

```diff
@@ -0,0 +1,32 @@
+package relayer
+
+import (
+	"github.com/dymensionxyz/roller/cmd/consts"
+	roller_utils "github.com/dymensionxyz/roller/utils"
+)
+
+func (r *Relayer) CheckClientsExist() (bool, error) {
+	rlyCfg, err := ReadRlyConfig(r.Home)
+	if err != nil {
+		return false, err
+
+	}
+	clientIDRollapp_raw, err := roller_utils.GetNestedValue(rlyCfg, []string{"paths", consts.DefaultRelayerPath, "dst", "client-id"})
+	if err != nil {
+		return false, nil
```

Comment:
better to return the err, and ignore it by the caller
keep same standard as in `GetActiveConnection`